### PR TITLE
Jasper / Rhys 

### PR DESCRIPTION
**Pull request by jaspermayone**

### CODE REVIEW

**Comment by jaspermayone** [comment]
Still need photos, working on that.

**Comment by jaspermayone** [comment]
@maxwofford we aim to have this ready for review tomorrow.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ _Application to Exhibit_
 **Brief description of you (or company)**
 _300 max characters_
 
+23,882 teen hackersteen hackers from around the world who code together.
```
Comment (maxwofford) [inline README.md:44]:
```suggestion
23,882 high school hackers from around the world who code together.
```

File: README.md [hunk 2]
```diff
@@ -72,39 +74,55 @@ _How would you explain your booth in fewest words possible? (World's Largest Key
...
 
-- Yes
-- No
+- [ ] Yes
+- [x] No
 
 **Do you have any special requests for your booth (please explain)**
 
 **Upload photos of what you're showcasing (limit to 4 photos)**
+HIGH RES PHOTOS IN ROOT OF REPO UNDER PHOTO FOLDER
```
Comment (maxwofford) [inline README.md:124]:
```suggestion
![](https://cloud-f04mirsa7-hack-club-bot.vercel.app/0assemble.jpg)
![](https://cloud-f04mirsa7-hack-club-bot.vercel.app/1daysofservice.jpeg)
```

File: README.md [hunk 3]
```diff
@@ -72,39 +74,55 @@ _How would you explain your booth in fewest words possible? (World's Largest Key
 
 _Explain your booth with many words._
 
+Hack Club is a nonprofit network of high school coding clubs and makers around the world. We will be showing a slection of highlights from the Hack Club Community, including You Ship, We Ship (YSWS) projects (various grants we give to teens), HCB our fiscal sponsorship platform, as well as presenting our clubs program. We will also be showcasing Hack Club's Day's of Service program. Days of Service is Hack Club’s initiative to support girls learning to code. By working with local youth organizations, high school Hack Club leaders run one-day coding events (a day of service) that help girls and gender minorities write their first lines of code. They might build their first ever website, video game, application and even code a robot arm to draw!
```
Comment (maxwofford) [inline README.md:77]:
```suggestion
Hack Club is a nonprofit network of high school coding clubs and teenage makers around the world. We'll bring a couple projects built by teenagers ranging from a PCB tilt detector to an open-source pen plotter.

In addition to the technical projects we have, we'll pass out sticker and posers for a few of our programs:
- Days of Service (https://daysofservice.hackclub.com/) is an initiative to support girls learning to code
- HCB (hackclub.com/hcb) is a fiscal sponsor for high school led projects
```

**Review by maxwofford — CHANGES_REQUESTED** [review]

**Comment by maxwofford** [comment]
Looking great, requested a few changes

**Comment by jaspermayone** [comment]
Finding club photo and YSWS photo now.

**Comment by jaspermayone** [comment]
@maxwofford let me know if you want me to actually submit this once it's ready.

**Pull request merged**